src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -43,6 +48,10 @@ internal namespace using the "{namespace}/" prefix.
 The REPLICA-CORE column displayed in the output table contains a list of
 replicas assignments in the form of: <Node-ID>-<Core>.
 
+If the DISABLED column contains a '-' value, then it means you are running this
+command against a cluster that is outdated and does not support the underlying 
+API.
```

Comment:
`against an older cluster that does not support`, or maybe even drop `older`. I don't think outdated has the best connotation here.